SWARAM-UI-Documentation/index.html

```diff
@@ -0,0 +1,460 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="components.css">
+    <link rel="stylesheet" href="index.css">
+
+    <title>Document</title>
```

Comment:
You can change the default title to your library name.

Reply:
I have changed it now